jaxrs/src/main/java/io/opentracing/contrib/jaxrs/client/SpanDecorator.java

```diff
@@ -0,0 +1,34 @@
+package io.opentracing.contrib.jaxrs.client;
+
+import javax.ws.rs.client.ClientRequestContext;
+import javax.ws.rs.client.ClientResponseContext;
+
+import io.opentracing.Span;
+import io.opentracing.contrib.jaxrs.URLUtils;
+import io.opentracing.tag.Tags;
+
+/**
+ * @author Pavol Loffay
+ */
+public interface SpanDecorator {
```

Comment:
I've made a type similar to this. it tends to only need to do tags. something to think about, but constraining to tags might be tighter